network/model.go

```diff
@@ -8,116 +8,57 @@ type CreateInteractionRequest struct {
 	NPCType       model.PlayerID `json:"npc_type,omitempty"`
 }
 
-// TODO figure out the minimum info the client will need
 type GetGameResponse struct {
-	ID              model.GameID                         `json:"id"`
-	Players         []Player                             `json:"players"`
-	PlayerColors    map[model.PlayerID]model.PlayerColor `json:"player_colors,omitempty"`
-	CurrentScores   map[model.PlayerColor]int            `json:"current_scores"`
-	LagScores       map[model.PlayerColor]int            `json:"lag_scores"`
-	Phase           model.Phase                          `json:"phase"`
-	BlockingPlayers map[model.PlayerID]model.Blocker     `json:"blocking_players,omitempty"`
-	CurrentDealer   model.PlayerID                       `json:"current_dealer"`
-	Hands           map[model.PlayerID][]model.Card      `json:"hands,omitempty"`
-	Crib            []model.Card                         `json:"crib,omitempty"`
-	CutCard         model.Card                           `json:"cut_card"`
-	PeggedCards     []model.PeggedCard                   `json:"pegged_cards,omitempty"`
-}
+	ID              model.GameID              `json:"id"`
+	Players         []Player                  `json:"players"`
+	PlayerColors    map[model.PlayerID]string `json:"player_colors,omitempty"`
+	CurrentScores   map[string]int            `json:"current_scores"`
+	LagScores       map[string]int            `json:"lag_scores"`
+	Phase           string                    `json:"phase"`
+	BlockingPlayers map[model.PlayerID]string `json:"blocking_players,omitempty"`
+	CurrentDealer   model.PlayerID            `json:"current_dealer"`
+
+	// Scrub out the other players' hands when returning this
```

Comment:
these scrub comments look like they're either done or on their way to becoming done. Time to remove?